isis/tests/FunctionalTestsStd2Isis.cpp

```diff
@@ -0,0 +1,227 @@
+#include <iostream>
+#include <sstream>
+
+#include <QIODevice>
+#include <QTextStream>
+#include <QStringList>
+#include <QFile>
+
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Cube.h"
+#include "Histogram.h"
+
+#include "gmock/gmock.h"
+
+#include "std2isis.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/std2isis.xml").expanded();
+
+TEST_F(TempTestingFiles, FunctionalTestsStd2IsisDefault) {
+  QString outCubeFileName = tempDir.path() + "/outTemp.cub";
+  
+  QVector<QString> args = {"from=data/stdFormatImages/rgb.png", "to="+outCubeFileName, };
```

Comment:
Will this trailing comma cause any issues?

Reply:
don't think so, at least it has caused no issues but I might as well remove it.